Sources/TuistLoader/Models+ManifestMappers/Config+ManifestMapper.swift

```diff
@@ -47,3 +51,22 @@ extension TuistCore.Config.GenerationOption {
         }
     }
 }
+
+private extension TuistCore.PluginLocation {
```

Comment:
To be consistent with other models, this should be in `PluginLocation+ManifestMapper.swift`.

Reply:
I'll refactor.